scania-digital-design-system/tegel
# fix(css): headline-05 updated to correct value

**Describe pull-request**  
Updated values for headline-05 to correct values according to figma.

**Solving issue**  
_In case of GitHub issue add # plus the number of the issue (for example #123) OR if it is Azure then AB# and number of ticket_
Fixes: [CDEP-3034](https://tegel.atlassian.net/browse/CDEP-3034)

**How to test**  
1. Go to storybook
2. Check so that headline-05 text is 16px instead of 14px

**Suggested test steps**

**Additional context**  
We need to check that modal title still wants to have tds-headline-05 as class after this change.


[CDEP-3034]: https://tegel.atlassian.net/browse/CDEP-3034?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
